web-services/security/src/main/java/datawave/security/user/UserOperationsBean.java

```diff
@@ -109,10 +124,39 @@ public AuthorizationsListBase listEffectiveAuthorizations() {
             DatawavePrincipal datawavePrincipal = (DatawavePrincipal) p;
             name = datawavePrincipal.getShortName();
             
+            // if we have any remote services configured, merge those authorizations in here
+            if (includeRemoteServices && remoteServices != null && !remoteServices.isEmpty()) {
```

Comment:
Consider `CollectionUtils`?
```suggestion
            if (includeRemoteServices && !CollectionUtils.isEmpty(remoteServices)) {
```